### ISSUE TITLE
Add History API test

### ISSUE THREAD

**Issue by jfrazelle** [docker/docker#12284]
https://github.com/docker/docker/pull/12260/files deleted a test, this needs to be reimplemented in integration-cli

**Comment by jfrazelle** [comment]
related to #12151

**Comment by WhoBrokeTheBuild** [comment]
I'll gladly take this

**Comment by thaJeztah** [comment]
@WhoBrokeTheBuild (awesome github handle btw); thanks! Be sure to include `closes #12284` in the description of you pull request .

**Comment by draghuram** [comment]
Based on comments in #12260, it is not clear to me if integration test is necessary. Are we sure that we do need such a test? If so, I would like to take a stab at it assuming @WhoBrokeTheBuild is fine with it.

**Comment by thaJeztah** [comment]
@draghuram let's give @WhoBrokeTheBuild a small bit of time to respond, just in case, if he doesn't, you can pick it.

**Comment by WhoBrokeTheBuild** [comment]
@draghuram Go right ahead, I haven't been able to get the tests to pass on a clean pull so I haven't exactly gotten very far with this

**Comment by draghuram** [comment]
Thanks. I will work on it.

**Comment by draghuram** [comment]
@jfrazelle I have updated my docker source and ran the full suite of tests both with in the development container and from the host itself. In both cases, the tests failed with the following error message:

    Test failures in ./daemon/graphdriver/devmapper"

I then tried to run `hack/make.sh test-integration-cli`, that too failed with some errors. I can provide more details if required but are there known issues with the tests at the moment? I just want to start with a clean slate before I add a new test. I use Ubuntu 14.04 as host, BTW. Thanks.
